experimental/PartitionedPermutations/src/SetPartitionFunctions.jl

```diff
@@ -0,0 +1,104 @@
+"""
+    number_of_blocks(V::SetPartition)
+
+Return the number of blocks of a `SetPartition`.
+
+# Examples
+```jldoctest
+julia> number_of_blocks(SetPartition([1, 2, 3], [2, 1, 3, 3]))
+3
+```
+"""
+function number_of_blocks(V::SetPartition)
+    # obtain one vector describing the partition V
+    vec = vcat(V.upper_points, V.lower_points)
+
+    # return the maximum number in vec
+    if length(vec) != 0
+        return maximum(vec)
+    else
+        return 0
+    end
+end
+
+"""
+    <=(V::SetPartition, W::SetPartition)
+
+Check if the set partition `V` is dominated by the set partition `W`. This is the case if every block of `V`
+is contained in exactly one block of `W`.
+
+# Examples
+```jldoctest
+julia> SetPartition([1, 1, 2], [1, 2, 3]) <= SetPartition([1, 1, 2], [1, 2, 1])
+true
+```
+"""
+function <=(V::SetPartition, W::SetPartition)
```

Comment:
Overloading `<=` seems dangerous and unexpected to me. Something like `is_dominated_by` seems more suitable to me

This function seems to belong to the `SetPartition` module instead.